examples/kafka/src/main/scala/org/apache/gearpump/streaming/examples/kafka/wordcount/KafkaWordCount.scala

```diff
@@ -59,6 +62,9 @@ object KafkaWordCount extends App with ArgumentsParser {
 
   val config = parse(args)
   val context = ClientContext(config.getString("master"))
+
+  implicit val system = context.system
```

Comment:
There are comment about akka serialization in definition of withValue and getValue:
https://github.com/clockfly/gearpump/blob/refactor_user_config/core/src/main/scala/org/apache/gearpump/cluster/UserConfig.scala